.github/workflows/lint.yml

```diff
@@ -13,16 +13,16 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v4
+        uses: actions/checkout@44c2b7a8a4ea60a981eaca3cf939b5f4305c123b
```

Comment:
Can you comment the version we _think_ we have fixed to, please? Makes it a little more human readable

Reply:
Sure, will do - thanks 👍

Reply:
All fixed up @batesyuk